### DISCUSSION TITLE
Risks of using VLESS without TLS

### DISCUSSION THREAD

**Discussion by APT-ZERO** [XTLS/Xray-core#3400] [Q&A] [answered]
Hello, what are the risks of using VLESS (TCP / WS) without TLS
as VLESS have no encryption like VMESS, so MITM Attacker can inspect the proxied traffic
But I have heard they can get our UUID and abuse our server too?

**Comment by mmmray** [comment, marked as answer]
all of that is correct. you can check it with wireshark yourself. everything on this page is in cleartext without TLS: https://xtls.github.io/en/development/protocols/vless.html
